File: lua/1145.lua
local x, y = io.read("*n", "*n")
local count = 0

for i = 1, y do
    count = count + 1
    io.write(i)

    if count == x then
        count = 0
        print()
    elseif i ~= y then
        io.write(" ")
    end
end

File: go/1145.go
package main

import "fmt"

func main() {
  var x, y, count int
  fmt.Scanf("%d %d", &x ,&y)

  for i := 1; i <= y; i++ {
    count += 1
    fmt.Printf("%d", i)

    if count == x {
      count = 0
      fmt.Println()
      continue
    }

    if i != y {
      fmt.Printf(" ")
    }
  }
}
